app/assets/stylesheets/account/profiles.scss

```diff
@@ -0,0 +1,3 @@
+.user_avatar {
+  display: inline-block!important;
```

Comment:
why do you need `!important` here?

Reply:
because i couldn't align "Add a profile photo"

Reply:
bulma has http://prntscr.com/lfqz94
so rework wrapper or create partial and render here

and put it on center (and photo also on center)
it must be consistent with other inputs